eng/scripts/Export-API.ps1

```diff
@@ -21,6 +21,7 @@ if ($SpellCheckPublicApiSurface) {
     npx cspell lint `
         --config "$PSScriptRoot/../../.vscode/cspell.json" `
         --no-must-find-files `
+        --root "$PSScriptRoot/../../" `
```

Comment:
FYI @danieljurek

Reply:
Without this, the root is the current directory, so if you run this in, say, your service directory (common) the `"overrides"` section of _.github/cspell.json_ won't match the file patterns.

Reply:
Thanks! I've been running all of these from the root of the repo.